### DISCUSSION TITLE
posts/Adjustment_basline_covariates/

### DISCUSSION THREAD

**Discussion by giscus** [Tea-Tasting-Statisticians/Tea-Tasting-Statisticians.github.io#5] [Q&A]
# posts/Adjustment_basline_covariates/

조주희 교수님이 진행하시는 advanced designs of clinical trials 수업에서 adjustment of baseline covariates에 대해 다루었다. 보통 임상시험에서는 공변량 (covariates)을 통제하지 않고 결과만 비교하기도 한다. 왜냐하면 randomization 결과로 confounder가 통제되고, collider와 같은 것들을 고려할 필요가 없기 때문이다.

https://tea-tasting-statisticians.github.io/posts/Adjustment_basline_covariates/

**Comment by shstat1729** [comment]
Sample size를 계산할 때에도 baseline covariates를 보정하여 계산할 수 있습니다. 만약 보정의 효과가 크다면 (variance를 크게 줄인다면), 더 적은 sample size로도 연구를 진행할 수 있을 것입니다.
